javascript-methods/main.js

```diff
@@ -0,0 +1,72 @@
+var num1 = 2;
+
+var num2 = 4;
+
+var num3 = 6;
+
+var maximumValue = Math.max(num1, num2, num3);
+
+var heroes = ['Spiderman', 'Batman', 'One Punch Man', 'Blue Ranger'];
+
+var randomNumber = Math.random(1, 100);
+
+randomNumber = randomNumber * heroes.length;
+
+var randomIndex = Math.floor(randomNumber);
+
+var randomHero = heroes[randomIndex];
+
+var library = [
+  {
+    title: 'Harry Potter and the Prisoner of Azkaban',
+    author: 'J.K Rowling'
+  },
+
+  {
+    title: 'Discworld',
+    author: 'Terry Pratchett'
+  }
+];
+
+var lastBook = library.pop();
+
+var firstBook = library.shift();
+
+var js = {
+  title: 'JavaScript for Impatient Programmers',
+  author: 'Dr. Axel Rauschmayer'
+};
+var css = {
+  title: 'CSS Secrets',
+  author: 'Lea Verou'
+};
+
+library.push(js);
+
+library.unshift(css);
+
+library.splice(1, 0);
```

Comment:
the second argument should be a 1!

Reply:
![javascript-methods](https://user-images.githubusercontent.com/90743100/155060088-e8550ca8-a4d9-4769-8429-cbe3a5701408.png)

Updated screenshot